pkg/controller/deployment/deployment_controller.go

```diff
@@ -634,3 +658,16 @@ func (dc *DeploymentController) updateDeployment(deployment *extensions.Deployme
 	// TODO: Using client for now, update to use store when it is ready.
 	return dc.expClient.Deployments(deployment.ObjectMeta.Namespace).Update(deployment)
 }
+
+// byCreationTimestamp sorts a list by creation timestamp, using their names as a tie breaker.
+type byCreationTimestamp []*api.ReplicationController
```

Comment:
I send a PR to eliminate the ByCreationTimeStamp duplication. see https://github.com/kubernetes/kubernetes/pull/21059